### PR TITLE
small howto for adding new api gateway lambdas

### PR DESCRIPTION
**Pull request by johnduffell**
@pvighi @jacobwinch @paulbrown1982 @twrichards 
I have made a start, hopefully at about the right level of detail.  feel free to make some more quick edits/commits to this PR directly, at least it's a start.  Not sure whether we should use google docs for these for ease of editing, or stick with github to keep it together.  I can replace this doc with a link if we think it would be better.

### CODE REVIEW

File: handlers/HOWTO-create-lambda.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+1. look at the [custom domain name](https://eu-west-1.console.aws.amazon.com/apigateway/home?region=eu-west-1#/custom-domain-names) that was created by the deploy
+1. look at the target domain name in the regional box (should start d-4765437.execute-api.eu-west-1.amazonaws.com or similar)
+1. edit the ApiGatewayTargetDomainName to point to the target domain name
+1. do a PR with the PROD still showing tbc and get it merged
+1. deploy to prod as below, then set up auto deployment in riffraff for the new project
+1. follow the steps after the "deploy to code" regarding the PROD domain name.
+1. go to runscope and set up a health check job
+
+## Deploying a lambda with riffraff for the first time
+1. preview the deploy using riffraff and only tick the "copy file to s3" step before hitting deploy
```
Comment (jacobwinch) [inline handlers/HOWTO-create-lambda.md]: 👍

**Comment by jacobwinch** [comment]
Looks good - I've tweaked a few things 👍

**Comment by johnduffell** [comment]
thanks jacob that's great, I'll merge this and we can add to it in future!

**Comment by twrichards** [comment]
👍 great stuff - nice and concise (_sorry for the delay in +1_)

**Pull request merged**